### ISSUE TITLE
Double-slash in the middle of the pattern is not collapsed

### ISSUE THREAD

**Issue by mgol** [mrmlnc/fast-glob#330]
### Environment

  * OS Version: macOS 11.5.1 (20G80)
  * Node.js Version: v14.17.4

### Actual behavior
```js
console.log(fastGlob.sync('/app/*.json'));  // ["/app/package.json"]
console.log(fastGlob.sync('/app//*.json')); // []
```

### Expected behavior
```js
console.log(fastGlob.sync('/app/*.json'));  // ["/app/package.json"]
console.log(fastGlob.sync('/app//*.json')); // ["/app/package.json"]
```

### Steps to reproduce

  1. See The code samples or open https://runkit.com/mgol/610aaad4e1d4410013466894. The `glob` package behaves as expected here.

### Code sample

Code already provided above.

**Comment by mrmlnc** [comment]
#### Why do I think this is a bug?

The following implementations support this form of patterns:

1. The original `glob` package;
1. `ls fixtures//*`;
1. python (`glob.glob('fixtures//*')`);
1. golang (`filepath.Glob("fixtures//*")`);

**Comment by mrmlnc** [comment]
Fixed in the `master` branch by #339. The release with `3.2.8` version will be released soon.
